packages/nextjs/src/config/webpack.ts

```diff
@@ -58,12 +56,6 @@ export function constructWebpackConfigFunction(
   const newWebpackFunction = (incomingConfig: WebpackConfigObject, buildContext: BuildContext): WebpackConfigObject => {
     let newConfig = { ...incomingConfig };
 
-    // if we're building server code, store the webpack output path as an env variable, so we know where to look for the
-    // webpack-processed version of `sentry.server.config.js` when we need it
-    if (newConfig.target === 'node') {
-      storeServerConfigFileLocation(newConfig);
```

Comment:
We can delete this function from utils now right?

Reply:
Yeah, it's gone. All of `utils.ts` is gone. That's why it's in red. 😛

Reply:
This is what I get for reviewing PRs on mobile lol